### PR TITLE
Adds ignore_dbs to configs, and makes the models respect it.

### PR DESCRIPTION
**Pull request by ed-**
RESPECT.

### CODE REVIEW

File: reddwarf/guestagent/db/models.py [hunk 1]
```diff
@@ -276,10 +278,16 @@ def __init__(self):
...
         return self._name
 
+    def _is_valid(self, value):
+        return value.lower() not in self._ignore_dbs
+
     @name.setter
     def name(self, value):
-        if not value or not self.dbname.match(value) or \
-                        string.find("%r" % value, "\\") != -1:
+        if any([not value,
```
Comment (rnirmal) [inline reddwarf/guestagent/db/models.py:286]: :) I see you got your 'any' in here.

**Comment by rnirmal** [comment]
+1

File: reddwarf/guestagent/db/models.py (hunk 1 above)
Comment (ed-) [inline reddwarf/guestagent/db/models.py:286]: Use your tools!

**Comment by jcru** [comment]
+1

**Comment by hub-cap** [comment]
RESPECT. settle down ali g

**Comment by hub-cap** [comment]
+1

**Comment by cp16net** [comment]
+1

**Pull request merged**